src/main/scala/com/lightbend/kafkalagexporter/AppConfig.scala

```diff
@@ -127,36 +136,22 @@ final case class KafkaCluster(name: String, bootstrapBrokers: String,
      """.stripMargin
   }
 }
-final case class AppConfig(pollInterval: FiniteDuration, lookupTableSize: Int, clientGroupId: String,
-                           clientTimeout: FiniteDuration, clusters: List[KafkaCluster], strimziWatcher: Boolean,
-                           metricWhitelist: List[String],
-                           prometheusConfig: Option[PrometheusConfig],
-                           graphiteConfig: Option[GraphiteConfig]) {
+
+final case class AppConfig(pollInterval: FiniteDuration, lookupTableSize: Int, sinkConfigs: List[SinkConfig], clientGroupId: String,
+                           clientTimeout: FiniteDuration, clusters: List[KafkaCluster], strimziWatcher: Boolean) {
   override def toString(): String = {
-    val graphiteString =
-      graphiteConfig.map { graphite => s"""
-        |Graphite: 
-        |  host: ${graphite.host}
-        |  port: ${graphite.port}
-        |  prefix: ${graphite.prefix}
-        """.stripMargin }.getOrElse("")
-    val prometheusString =
-      prometheusConfig.map { prometheus => s"""
-        |Prometheus: 
-        |  port: ${prometheus.port}
-        """.stripMargin }.getOrElse("")
     val clusterString =
       if (clusters.isEmpty)
         "  (none)"
       else clusters.map(_.toString).mkString("\n")
+    val sinksString = sinkConfigs.mkString("")
     s"""
        |Poll interval: $pollInterval
        |Lookup table size: $lookupTableSize
-       |Metrics whitelist: [${metricWhitelist.mkString(", ")}]
+       |Metrics whitelist: [${sinkConfigs.head.metricWhitelist.mkString(", ")}]
        |Admin client consumer group id: $clientGroupId
        |Kafka client timeout: $clientTimeout
-       |$prometheusString
-       |$graphiteString
```

Comment:
Something funny happened here. The metrics whitelist should remain unchanged and you can iterate the new `sinkCconfigs` to replace the existing prometheus and graphite configs.

Reply:
Can you please clarify ? 

All the sinks will have the same whitelists. We just need to fetch from one of the sinks (we use head of the list) to get the whitelist.

Reply:
Got it.